frontend/src/container/QueryBuilder/filters/AggregateEveryFilter/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Input } from 'antd';
+import getStep from 'lib/getStep';
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { AppState } from 'store/reducers';
+import { IBuilderQueryForm } from 'types/api/queryBuilder/queryBuilderData';
+import { GlobalReducer } from 'types/reducer/globalTime';
+
+import { selectStyle } from '../QueryBuilderSearch/config';
+
+function AggregateEveryFilter({
+	onChange,
+	query,
+}: AggregateEveryFilterProps): JSX.Element {
+	const { maxTime, minTime } = useSelector<AppState, GlobalReducer>(
+		(state) => state.globalTime,
+	);
+
+	const stepInterval = useMemo(
+		() =>
+			getStep({
+				start: minTime,
+				end: maxTime,
+				inputFormat: 'ns',
+			}),
+		[maxTime, minTime],
+	);
+
+	const handleKeyDown = (event: {
+		keyCode: number;
+		which: number;
+		preventDefault: () => void;
+	}): void => {
+		const keyCode = event.keyCode || event.which;
+		const isBackspace = keyCode === 8;
+		const isNumeric =
+			(keyCode >= 48 && keyCode <= 57) || (keyCode >= 96 && keyCode <= 105);
+
+		if (!isNumeric && !isBackspace) {
+			event.preventDefault();
+		}
+	};
+
+	return (
+		<Input
+			type="text"
```

Comment:
can you please confirm the validation here i am able to enter text if we are entering in seconds only https://ant.design/components/input-number

Reply:
Its number input with arrow key, @palashgdev 
If we use the above component it will show an up and down arrow key. which is not present in UI.
So, I did validation of the number only with type = text.